apps/aecore/lib/aecore/oracle/oracle.ex

```diff
@@ -294,21 +237,40 @@ defmodule Aecore.Oracle.Oracle do
     end
   end
 
+  @spec get_registered_oracles :: map()
+  def get_registered_oracles do
+    otree = Chain.chain_state().oracles.otree
+    keys = PatriciaMerkleTree.all_keys(otree)
+
+    registered_oracles_key =
+      Enum.reduce(keys, [], fn x, acc ->
+        if byte_size(x) == 33 do
```

Comment:
please don't use magic numbers, move it to a named constant https://en.wikipedia.org/wiki/Magic_number_(programming)